mne/annotations.py

```diff
@@ -608,4 +614,11 @@ def events_from_annotations(raw, event_id=None, regexp=None,
     logger.info('Used Annotations descriptions: %s' %
                 (list(event_id_.keys()),))
 
+    if len(dropped) > 0:
+        dropped = list(set(dropped))
+        logger.info("{0} trigger(s) have been dropped, such as {1}. "
```

Comment:
I'm not sure that this is common to every annotation or only to brainvision. This has to do with this parameter removed from `events = _read_vmrk_events(mrk_fname, event_id, trig_shift_by_type)` now `_read_vmrk(mrk_fname)`:

```diff
-
-    event_id : dict | None
-    event_id : dict | None
-    event_id : dict | None
-         Special events to consider in addition to those that follow the normal
-         BrainVision trigger format ('###' with an optional single character
-         prefix). If dict, the keys will be mapped to trigger values on the
-         stimulus channel. Example: {'SyncStatus': 1; 'Pulse Artifact': 3}.
-         If None or an empty dict (default), only BrainVision format events are
-         added to the stimulus channel. Keys are case sensitive. "New Segment"
-         markers are always dropped.
```